app/util/data_preparation/confluence_prepare_data.py

```diff
@@ -21,17 +24,44 @@
 ENGLISH_GB = 'en_GB'
 
 
+def print_timing(message, sep='-'):
```

Comment:
This is a code duplication. jsm_prepare_data already has this. Need to extract this decorator to a common place, e.g. introduce new util file for prepare_data scripts.